src/php/NamespaceExtractor/Extractor/NamespaceInformation.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\NamespaceExtractor\Extractor;
+
+class NamespaceInformation
```

Comment:
Please, remember doing new classes [final](https://chemaclass.medium.com/final-classes-in-php-9174e3e2747e) according to our standard of closing classes by default in order to encourage composition over inheritance :)

Reply:
done